api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -506,6 +506,14 @@ message HttpConnectionManager {
   //
   // 3. Tracing decision (sampled, forced, etc) is set in 14th byte of the UUID.
   RequestIDExtension request_id_extension = 36;
+
+  // Determines if port part should be removed from host/authority header before any processing
+  // of request by HTTP filters or routing. Port would be removed only if it is equal to the listener's
```

Comment:
Port -> The port
and I'd change "connect" to CONNECT again just for consistency.